inventories/group_vars/control-plane/headscale.yaml

```diff
@@ -0,0 +1,16 @@
+---
+headscale_image_name: headscale/headscale
+headscale_image_tag: 0.16.4-alpine
```

Comment:
All images and image names come from the release repository, so they are not required in this file.